src/widgets/torrent/TorrentTile.tsx

```diff
@@ -98,6 +106,127 @@ function TorrentTile({ widget }: TorrentTileProps) {
+      accessorFn: (row) => (row.isCompleted ? 'true' : 'false'), //must be strings
+      id: 'isCompleted',
+      header: t('card.table.header.isCompleted'),
+      filterVariant: 'checkbox',
+      Cell: ({ cell }) =>
+        cell.getValue() === 'true' ? 'Completed' : 'Incomplete',
+      size: 220,
+    },
+    {
+      id: "dateAdded",
+      accessorFn: (row) => new Date(row.dateAdded),
+      header: t('card.table.header.dateAdded'),
+      sortDescFirst: true,
+      enableMultiSort: true,
+      Cell: ({ cell }) => new Date(String(cell.getValue())).toLocaleString(),
+      filterVariant: 'date-range',
+    },
+    {
+      accessorKey: 'name',
+      header: t('card.table.header.name'),
+      sortDescFirst: true,
+      enableMultiSort: true,
+    },
+    {
+      accessorKey: 'totalSize',
+      header: t('card.table.header.size'),
+      Cell: ({ cell }) => formatSize(Number(cell.getValue())),
+      sortDescFirst: true,
+      enableMultiSort: true,
+      filterVariant: 'range-slider',
+      filterFn: 'betweenInclusive',
+      mantineFilterRangeSliderProps: {
+        step: 1024,
+        label: (value: number) => formatSize(value),
+      }
+
+    },
+    {
+      accessorKey: 'uploadSpeed',
+      header: t('card.table.header.upload'),
+      Cell: ({ cell }) => formatSpeed(Number(cell.getValue())),
+      sortDescFirst: true,
+      enableMultiSort: true,
+      filterVariant: 'range-slider',
+      filterFn: 'betweenInclusive',
+      mantineFilterRangeSliderProps: {
+        max: 100 * 1024 * 1024, //100MiB/s
+        min: 0,
+        label: (value: number) => formatSpeed(value),
+      }
+    },
+    {
+      accessorKey: 'downloadSpeed',
+      header: t('card.table.header.download'),
+      Cell: ({ cell }) => formatSpeed(Number(cell.getValue())),
+      sortDescFirst: true,
+      enableMultiSort: true,
+      filterVariant: 'range-slider',
+      filterFn: 'betweenInclusive',
+      mantineFilterRangeSliderProps: {
+        max: 100 * 1024 * 1024, //100MiB/s
+        min: 0,
+        label: (value: number) => formatSpeed(value),
+      }
+    },
+    {
+      accessorKey: 'eta',
+      header: t('card.table.header.estimatedTimeOfArrival'),
+      Cell: ({ cell }) => formatETA(Number(cell.getValue())),
+      sortDescFirst: true,
+      enableMultiSort: true,
+      enableColumnFilter: false,
+    },
+  ], []);
+
+  const torrentsTable = useMantineReactTable({
+    columns,
+    data: filteredTorrents,
+    enablePagination: false,
+    enableBottomToolbar: false,
+    enableMultiSort: true,
+    enableColumnActions: false,
+    enableColumnFilters: false,
+    enableSorting: true,
+    initialState: {
+      showColumnFilters: false,
+      showGlobalFilter: false,
+      density: 'xs',
+      sorting: [{ id: 'dateAdded', desc: true }],
+      columnVisibility: {
+        isCompleted: false,
+        dateAdded: false,
+        name: true,
+        totalSize: true,
+        // uploadSpeed: width > MIN_WIDTH_MOBILE,
```

Comment:
What is with this commented out code?

Reply:
In initialState, it doesn't correctly compare : width seems to be egal to 0 so I softly removed it to keep a track of this try